scarb/src/bin/scarb/commands/commands.rs

```diff
@@ -1,9 +1,159 @@
 use anyhow::Result;
+use serde::{Serialize, Serializer};
+use std::collections::BTreeMap;
+use std::fmt::Display;
+use std::path::PathBuf;
+use std::{env, fs};
 
+use crate::args::ScarbArgs;
 use scarb::core::Config;
+use scarb::process::is_executable;
+use scarb::ui::Message;
+use scarb::EXTERNAL_CMD_PREFIX;
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Debug, Serialize)]
+pub enum CommandInfo {
```

Comment:
```suggestion
enum CommandInfo {
```

not used outside this file?